nautobot_golden_config/utilities/helper.py

```diff
@@ -46,21 +47,27 @@ def get_job_filter(data=None):
     elif data.get("device"):
         query.update({"id": data["device"].values_list("pk", flat=True)})
 
-    base_qs = Device.objects.none()
+    raw_qs = Q()
```

Comment:
This is getting beyond my knowledge as to why this is preferred. Can you explain in human terms what makes this a better solution?

Reply:
Q objects are meant for items like AND and OR operators in a queryset. This is also taking advantage of some of the improvements in 1.4 for DynamicGroups.